api/routes/routes.js

```diff
@@ -1,3 +1,14 @@
+const budgetControllers = require('../controllers/budgetControllers');
+const categoryControllers = require('../controllers/categoryControllers');
+const expenseControllers = require('../controllers/expenseControllers');
+
 module.exports = app => {
-  // Todo: Fill in your routes here
+  app.route('/budget')
+    .post(budgetControllers.createBudget);
+  app.route('/category')
+    .get(categoryControllers.listCategories)
+    .post(categoryControllers.createCategory);
+  app.route('/expense')
+    .get(expenseControllers.listExpenses)
+    .post(expenseControllers.createExpense);
 };
```

Comment:
Overall, really clean code! I'm happy to have been able to take a look! Just double check those commit messages and make them more meaningful so your future self will thank you :)